### DISCUSSION TITLE
Issues verifiying my enrollment.

### DISCUSSION THREAD

**Discussion by metadruid** [community/community#54917] [GitHub Education]
### Hi everyone,

I was going through the process of verifying my enrollment but the college I attend does not give out student IDs and GitHub had asked for that, saying that I could not use any other form of verification. Since I did not have this, I sent them a copy of my enrollment forms that had been signed by my advisor, and in response, I got an automatic email that I was unlikely to be accepted because I had not used my devices camera and my GitHub account was not filled out fully? I'm not sure why they are saying this, because I had to take the photo on my phone. I even went as far as to include my ID with sensitive information blocked with paper. 

The enrollment verification form is dated and has my signature and my advisors signature, as well as my full name and phone number (which is linked to my student email). This is quite literally the best that I can do without a student ID, and I believe that the proof is beyond what should be required. I don't mean to come off as rude, this is just very frustrating for me.

**Comment by samus-aran** [comment]
Hi @richoff0days 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and I'll be happy to assist!
